### PR TITLE
MRG: Project 2 Path Report 

### PR DESCRIPTION
**Pull request by jcpalumbo3**
Editing the final report documentation for the path subproject. Let me know if there's anything you'd like to include. 
@sicihuang @alannaiverson @jJasonWang @aksam-ahmad @togawa28

### CODE REVIEW

File: doc/source/report/path.rst [hunk 1]
```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
...
+simply be a mouse shifting weight from one foot to the other and back again.
+Since this is not the type of path we are interested in analyzing, it makes
+sense to filter them out. The remove duplicates function deals with an issue
+in the data where the same x and y coordinates appear in adjacent rows but
+with different timestamps. This caused a problem when computing the angles of
+the paths so, while it is a trivial removal, it is necessary.
+
+**Step 3**: Choose Key Features
+
+The path features we wrapped up into functions are as follows:
```
Comment (sicihuang) [inline doc/source/report/path.rst]: This line was italicized for some reason. Maybe you want to check rst file style?

File: doc/source/report/path.rst [hunk 2]
```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
...
+**Step 4**:  Interpretation
+
+Below are the plots generated based on the features we calculated above, per
+strain, mouse, and day. With these plots, we can draw some initial conclusions
+about differences between strains in the results section.
+
+Results
+-------------------------
+
+.. figure:: figure/dist_path.png
```
Comment (sicihuang) [inline doc/source/report/path.rst]: We should comment on these plots. For example:
"Strain 0 appeared to be the most active overall. For all mice in this strain, activeness peaked around days 6-8 and fell after that. In contrast, there is a huge variation in the activeness among mice of strain 1."

**Comment by aksam-ahmad** [comment]
**Further Studies**

In the future, we would like to explore the following areas:

- Further data smoothing: We would like to further develop a robust methodology to detect points which could be deemed ‘measurement noise’. Measurement noise can be defined as consecutive observations which indicate high-turning angles (i.e., above 150 degrees) over very short time intervals. Such observations would skew results on number of sharp turns, average turning angles, etc. It is possible the measurement noise obscures the path behavior differences between strains.

- Path behavior as it relates to active state lifestyle: We would like to explore the influence of a mouse’s active state behaviors on its path behavior. A possible question we may be answer is the following: how does hunger and thirst affect the mouse’s path diversity? (i.e., does the mouse get sluggish after having eaten and therefore takes shorter, slower paths?)

- Obtaining optimal tuning parameters: We would like to determine which parameters would be optimal for analysis purposes. These parameters include, but are not limited to, the cutoff time for a path, the minimum path length (in observations) and turning angle threshold for noise detection. 

- Use path features for classification: We would like to use the features calculated within the functions and apply machine learning techniques to determine whether path features (speed, turning angles, distance traveled, etc.) can be used for classification.

**Comment by jcpalumbo3** [comment]
I added @aksam-ahmad 's future studies section, updated the results, and added additional remarks. I think after adding @togawa28 results, we will just one final check.

**Comment by jcpalumbo3** [comment]
@jarrodmillman This is ready to merge. Coverage went down slightly, but the issue doesn't seem to be related to the rst file as far as we can tell.

**Comment by jarrodmillman** [comment]
@jcpalumbo3 Thanks, I will take it from here.

**Comment by jarrodmillman** [comment]
Rebased in #204

**Pull request closed without merging**